File: Provider/Neighborhood-Services/Neighborhood.Services.Application/Messages/DTOs/MessageSelectedDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Neighborhood.Services.Application.Messages.DTOs
{
    public class MessageSelectedDto
    {
        public int conversationId { set; get; }
        public int messageId { set; get;  }
        public string content { set; get; }

        public string senderId { set; get;  }
        public string senderName { set; get; }
        public bool read {  set; get; }
        public bool deleted { set; get; }

        public DateTime sentAt { set; get; } = DateTime.UtcNow;

    }
}

File: Client/neighborhood.services.client/src/app/core/models/message-selected-dto.ts
export class MessageSelectedDto {

public messageId?: number;
public conversationId?: number;
public senderId?: string
public senderName?: string;
public content?: string;
public read?: boolean;
public deleted?: boolean;
public sentAt: Date = new Date();


    // public id?: number;
    // public senderId?: string;
    // public content!: string;
    // public bookingId!: number;

constructor(init?: Partial<MessageSelectedDto>) {
    Object.assign(this, init);
  }
}

